src/rp2_common/pico_aon_timer/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ pico_mirrored_target_link_libraries(pico_aon_timer INTERFACE pico_util)
 
 if (TARGET hardware_rtc)
     pico_mirrored_target_link_libraries(pico_aon_timer INTERFACE hardware_rtc)
```

Comment:
note i think you still need `hardware_clocks` here

Reply:
hardware_rtc depends on hardware_clocks? Do I need to repeat it here?